assets/app/data/crowbar/services/upgrade-status.factory.js

```diff
@@ -50,19 +50,36 @@
          *     API will not trigger `onError` handler and will not stop polling. The downtime
          *     allowance is common for whole call so if there are multiple short unavailability
          *     periods, the total time (sum) is checked.
+         * @param {function} [processPartialStatus=undefined] - If specified, will be executed each time with success
```

Comment:
To keep things consitent, I would name this something like `onRunning` and maybe move it together with other callbacks.
Since we already have multiple optional arguments, maybe this function should be converted to "object argument" form (i.e. one argument being an object with current arguments as properties)?